### DISCUSSION TITLE
Updated from 1.27 to 1.30 - Config Error

### DISCUSSION THREAD

**Discussion by indaleco** [dani-garcia/vaultwarden#4234] [Q&A]
I don't keep up on my install like I should.  I did a pull to the latest version and then I had an error that it couldn't load the configuration. 

I didn't dig too deep into it, I reverted to the snapshot I took before upgrading, but was curious if anyone else has seen this or if I just missed something stupid (which I probably did).  I'm not super linux or docker savvy and I didn't have time to dig into it and I didn't get a screen shot. 

I'll attempt it again and if I get it I will also share my results.

**Comment by BlackDex** [comment]
We need more information regarding the error. There shouldn't be anything blocking or breaking here.
